### PR TITLE
Update 09-supp-intro-rstudio.md

### PR DESCRIPTION
**Pull request by nrambergpihl**
I have a comment on the Introduction to RStudio episode of the Programming with R Software Carpentry Lesson. This episode provides a detailed overview of working in R Studio. However, no visuals are provided. Would it be possible to add either a screenshot of RStudio or a depiction of the RStudio working environment? For attendees of a Software Carpentry Session, the instructor can provide an overview during the lesson. However, for anyone who may be going through a lesson on their own or referring to the lesson as a reminder, there are no visuals on the episode page to help the learner follow along. The text does provide information on where each major component of RStudio exists in the working environment, i.e. "console window (in RStudio, the bottom left panel)." However, adding visual aids could be helpful for visual learners. See suggestions in the text for where visual aids could be included.

<details>
<summary><strong>Instructions</strong></summary>

Thanks for contributing! :heart:

If this contribution is for instructor training, please email the link to this contribution to
checkout@carpentries.org so we can record your progress. You've completed your contribution
step for instructor checkout by submitting this contribution!

Keep in mind that **lesson maintainers are volunteers** and it may take them some time to
respond to your contribution. Although not all contributions can be incorporated into the lesson
materials, we appreciate your time and effort to improve the curriculum. If you have any questions
about the lesson maintenance process or would like to volunteer your time as a contribution
reviewer, please contact The Carpentries Team at team@carpentries.org.

You may delete these instructions from your comment.

\- The Carpentries
</details>

### CODE REVIEW

**Comment by HaoZeke** [comment]
Hi @nrambergpihl. Thanks for the suggestion, it sounds great to me. Would you be willing/able to add screenshots in the positions you mentioned in the text?

File: _episodes/09-supp-intro-rstudio.md [hunk 1]
```diff
@@ -31,11 +31,11 @@ Let's start by learning about our tool.
 There are two main ways of interacting with R: using the console or by using
 script files (plain text files that contain your code).
 
-The console window (in RStudio, the bottom left panel) is the place where R is
+The console window (in RStudio, the bottom left panel) ***Insert visual aid here*** is the place where R is
```
Comment (nrambergpihl) [inline _episodes/09-supp-intro-rstudio.md:34]: RStudio_Screenshot_ConsoleWindow

File: _episodes/09-supp-intro-rstudio.md [hunk 2]
```diff
@@ -31,11 +31,11 @@ Let's start by learning about our tool.
...
 waiting for you to tell it what to do, and where it will show the results of a
 command.  You can type commands directly into the console, but they will be
 forgotten when you close the session. It is better to enter the commands in the
-script editor, and save the script. This way, you have a complete record of what
+script editor ***Insert visual aid here*** , and save the script. This way, you have a complete record of what
```
Comment (nrambergpihl) [inline _episodes/09-supp-intro-rstudio.md:38]: RStudio_Screenshot_ScriptingWindow

**Comment by nrambergpihl** [comment]
Hello @HaoZeke. Please see the screenshots that were added. I included a blue box around the windows in R Studio being referenced. I am happy to alter these as needed. Let me know if you need any further screenshots from me. Cheers!

**Comment by zkamvar** [comment]
I'm helping the current lesson Maintainers process outstanding pull requests on this repository, in preparation for [transition to the new lesson infrastructure](https://carpentries.github.io/workbench/faq.html#update-fork-from-styles).

Thank you for this contribution @nrambergpihl. The images look good, but they needed to be placed in the repository with markdown and alt text pointing to them. Because of this, I will be closing this PR without merging the change. I'm sorry for the delay in reviewing your contribution and hope that you will continue to contribute to The Carpentries lessons.

**Pull request closed without merging**